tests/integration_test.go

```diff
@@ -185,3 +201,20 @@ func TestDiffAndAnalysis(t *testing.T) {
 		})
 	}
 }
+
+func TestMain(m *testing.M) {
+	// setup
+	ctx := context.Background()
+	cli, _ := client.NewEnvClient()
```

Comment:
This needs to call https://github.com/GoogleCloudPlatform/container-diff/blob/master/cmd/root.go#L46-L54